internal/sync/sync.go

```diff
@@ -71,11 +71,13 @@ func NewSynchronizer(cfg *config.Sync, feederClient *feeder.Client, syncManager
 	synchro := new(Synchronizer)
 	synchro.logger = Logger.Named("Sync Service")
 	synchro.feeder = feederClient
-	if !cfg.Trusted {
-		ethereumClient, err := ethclient.Dial(cfg.EthNode)
-		if err != nil {
-			synchro.logger.Fatal("Unable to connect to Ethereum Client", err)
-		}
+	ethereumClient, err := ethclient.Dial(cfg.EthNode)
+	trusted := err != nil
+	if trusted {
+		// If we can't connect to an ethereum node, log an error
+		// and sync from the (trusted) feeder gateway.
+		synchro.logger.With("error", err).Error("Cannot connect to Ethereum Client: syncing state from the centralized feeder gateway")
```

Comment:
I don't think this should be an error. If the user did not provide an Ethereum node address, we should probably assume that was their intention and log that at `INFO` i.e. `defaulting to syncing from gateway`.

Reply:
Thanks! That would be much better. I've updated the code to incorporate your suggestions.